### DISCUSSION TITLE
Directive to guide the user to read software project standard.

### DISCUSSION THREAD

**Discussion by larssb** [PowerShellOrg/Plaster#386] [Ideas]
This is a follow up to comments in #5 from 170623.

- It would be great to have a directive that does one of the following suggestions, depending on what is possible:

1. Open the default browser on a users box, when processing the template is done, sending the user to a documentation URI, where the user can read up on the coding standard/software development standard.
2. Fetches documentation data via a REST or cURL call, to stay in the PowerShell world I guess invoke-webrequest, shows the data to the end-user. In VS Code the new HTML tab option could be used 👍 .....

- It might be worth considering that a user could get really tired of getting this prompt every time they scaffold from the template. However, the documentation on the other side, might have been updated....goes both ways....thinking out loud here. Feedback is welcome.

Thank you.

**Comment by PowerSchill** [comment]
How about just including a README.MD file? That would fit with current community standards.

**Comment by larssb** [comment]
@PowerSchill - I know what you are saying and it is a good suggestions. However, users that have access to or is familiar with IDE's and so forth might see it as uphill to update a ReadMe.md file in a repo.....but yes....thinking about it.
